### PR TITLE
Added basic specs

### PR DESCRIPTION
**Pull request by fedelcar**
# Summary

Created specs for some controllers
# Trello card

https://trello.com/c/sMAGYmIC

### CODE REVIEW

**Comment by fedelcar** [comment]
@epintos

File: spec/controllers/article_spec.rb [hunk 1]
```diff
@@ -1,24 +1,20 @@
 require 'rails_helper'
+require 'spec_helper'
 
 RSpec.describe ArticlesController do
   describe 'GET #index' do
     it 'responds successfully with an HTTP 200 status code' do
+      @user = User.create(:email    => "imanexample@example.com",:password => "shhhpassword")
```
Comment (epintos) [inline spec/controllers/article_spec.rb:7]: - I think you are not running rubocop in specs
- I recommend you using FactoryGirl gem

File: spec/controllers/article_spec.rb [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
+      
+      expect(response).to have_http_status(302)
     end
 
-    it 'renders the index template' do
-      get :index
-      expect(response).to render_template('index')
-    end
+    # it 'renders the index template' do
+    #   get :index
```
Comment (epintos) [inline spec/controllers/article_spec.rb:15]: What happened with this?

File: spec/controllers/comment_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe CommentsController do
+  describe 'GET a post' do
+    it 'responds successfully with an HTTP 200 status code' do
+
```
Comment (epintos) [inline spec/controllers/comment_spec.rb:7]: ?

**Pull request closed without merging**